src/elexclarity/convert.py

```diff
@@ -22,7 +23,18 @@ def convert(data, statepostal=None, level=None, outputType="results", style="def
         data = [xmltodict.parse(data, attr_prefix="")["ElectionResult"]]
 
     if level == "precinct" or level == "county":
-        county_fips_lookup = {v["name"]: k for k, v in STATE_COUNTIES[statepostal].items()}
+        # If specific county mapping is provided
+        if countymapping:
+            fips_mapping = countymapping[statepostal]
+            county_fips_lookup = {v["name"]: k for k, v in fips_mapping.items()}
+        # Else if fips mapping static data is available
+        elif fips_mapping_exists(statepostal):
```

Comment:
Rather than stashing mappings here, I was thinking we would generate an appropriate mapping using elex-static-data (in the data importer) and pass that in via `countymapping`. That way this repo will stay light-weight and we can still use our existing mappings (the Georgia mapping in particular is obviously not that large, but if we use this for other states the mapping will grow).